systemdspawner/systemdspawner.py

```diff
@@ -288,6 +288,10 @@ async def start(self):
                 for path in self.readwrite_paths
             ]
 
+        if len(self.unit_extra_properties) > 0:
+            for property, value in self.unit_extra_properties.items():
+                self.unit_extra_properties[property] = self._expand_user_vars(value)
+
```

Comment:
```suggestion
        for property, value in self.unit_extra_properties.items():
            self.unit_extra_properties[property] = self._expand_user_vars(value)

```

Reply:
I think this should be equivalent - no iteration is done if extra properties is empty.

Reply:
Thanks. Much better.